src/verifiers/sev-snp/crypto.c

```diff
@@ -0,0 +1,175 @@
+/* Copyright (c) 2022 Intel Corporation
+ * Copyright (c) 2020-2022 Alibaba Cloud
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <openssl/sha.h>
+#include <openssl/ts.h>
+#include <rats-tls/log.h>
+#include <string.h>
+#include "crypto.h"
```

Comment:
Usually, the headers should be arranged in this order:
```
// standard headers
#include <string.h>
// 3rd
#include <openssl/sha.h>
#include <openssl/ts.h>
// rats-tls specific
#include <rats-tls/log.h>
// local headers
#include "crypto.h"
```

Please follow this rule to arrange the order of headers.